rotkehlchen/db/dbhandler.py

```diff
@@ -502,6 +507,7 @@ def _connect(self, conn_attribute: Literal['conn', 'conn_transient'] = 'conn') -
             # switch to WAL mode: https://www.sqlite.org/wal.html
             conn.execute('PRAGMA journal_mode=WAL;')
         except sqlcipher.DatabaseError as e:  # pylint: disable=no-member
+            conn.close()
```

Comment:
can close fail?

Reply:
no i guess, in other places also we use it safely

Reply:
It seems that never fails https://github.com/python/cpython/blob/77133f570dcad599e5b1199c39e999bfac959ae2/Modules/_sqlite/connection.c#L469